.github/workflows/release.yml

```
@@ -35,6 +35,20 @@ jobs:
           checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .path')
           echo "::set-output name=hashes::$(cat $checksum_file | base64 -w0)"
 
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          check-latest: true
+      - uses: imjasonh/setup-ko@v0.6 # This installs the current latest release.
+      - run: |
+          tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
+          ko build --bare --platform=all -t latest -t ${{ github.sha }} -t ${tag} ./
```

```suggestion
      - uses: imjasonh/setup-ko@v0.6 # This installs the current latest release.
      - uses: sigstore/cosign-installer@v2.8.1
      - run: |
          tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
          img=$(ko build --bare --platform=all -t latest -t ${{ github.sha }} -t ${tag} ./)
          echo "built ${img}"
          cosign sign ${img} \
              -a sha=${{ github.sha }} \
              -a run_id=${{ github.run_id }} \
              -a run_attempt=${{ github.run_attempt }} \
              -a tag=${tag}
```

cribbed from here: https://github.com/ko-build/ko/blob/f158992ccb55617b98b8c580d7a7e25a2aaa3ae8/.github/workflows/image.yaml#L35-L38